frameworks/Java/sabina/src/test/java/sabina/benchmark/ApplicationTest.java

```diff
@@ -39,13 +38,13 @@
  * <p>TODO Change assert's order
  */
 public final class ApplicationTest {
-    private static final int THREADS = 16, EXECUTIONS = 32, WARM_UP = 32;
+    private static final int THREADS = 16, EXECUTIONS = 75, WARM_UP = 10;
 
     private static final String ENDPOINT = "http://localhost:5050";
     private static final Gson GSON = new Gson ();
 
     @BeforeClass public static void setup () {
-        main (null);
+        Application.main (null);
     }
 
     @BeforeClass public void warm_up () throws IOException {
```

Comment:
We also perform an initial warmup before we actually benchmark your application, this is probably redundant :-)